### DISCUSSION TITLE
Incorrect answer on Pimcore Junior Certification exam

### DISCUSSION THREAD

**Discussion by calebrowehdmz** [pimcore/pimcore#15202] [Q&A / Support]
I apologize if this is not the place for such a discussion. I recently ran into a question on the Pimcore Junior cert exam that seemed rather odd to me.

> What can be defined via permissions out of the box in Pimcore Admin Interface?
> - Field based permissions in data objects
> - Access rules for Pimcore elements via workspaces
> - Access rules based on property values of elements
> - Visible custom layouts and perspectives
> - Permissions to most of main menu entries

It's a multiple choice question, and I included "Field based permissions in data objects" in my answer. Turns out, that is wrong. Why? I just tested it, and you can create custom layouts for data objects that allow you to show/hide certain fields when viewing data objects.

Am I reading or understanding the question or answer wrong?

**Comment by Gintron** [comment]
I am struggling with this question too
